app/_posts/2020-06-18-h2o-resources-for-getting-started.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "H2O Resources for Getting Started"
+author: kelly-fitzpatrick
+date: 2020-06-18
+---
+---
```

Comment:
I think my suggestion may have added an additional line here! If you drop line 7 this is good. 
```suggestion
```